dotnet/test/support/Events/EventFiringWebDriverTest.cs

```diff
@@ -40,8 +40,14 @@ public class EventFiringWebDriverTest
         public void Setup()
         {
             mockDriver = new Mock<IWebDriver>();
-            mockElement = new Mock<IWebElement>();
-            mockShadowRoot = new Mock<ISearchContext>();
+            mockElement = new Mock<IWebElement>()
```

Comment:
The null checks found a hole in the mocking, these changes are necessary to make those tests pass.